src/pkg/errs/errs.go

```diff
@@ -3,35 +3,18 @@ package errs
 import (
 	"errors"
 
+	"github.com/alcionai/corso/src/pkg/errs/core"
 	"github.com/alcionai/corso/src/pkg/repository"
 	"github.com/alcionai/corso/src/pkg/services/m365/api/graph"
 )
 
-// expose enums, rather than errors, for Is checks.  The enum should
-// map to a specific internal error that can be used for the actual
-// errors.Is comparison.
-type errEnum string
-
-const (
-	ApplicationThrottled      errEnum = "application-throttled"
-	BackupNotFound            errEnum = "backup-not-found"
-	InsufficientAuthorization errEnum = "insufficient-authorization"
-	RepoAlreadyExists         errEnum = "repository-already-exists"
-	ResourceNotAccessible     errEnum = "resource-not-accesible"
-	ResourceOwnerNotFound     errEnum = "resource-owner-not-found"
-	ServiceNotEnabled         errEnum = "service-not-enabled"
-)
-
 // map of enums to errors.  We might want to re-use an enum for multiple
 // internal errors (ex: "ServiceNotEnabled" may exist in both graph and
```

Comment:
Now that we have moved around, we might wanna update the example mentioning `ServiceNotEnabled`.